### ISSUE TITLE
Flow confusing with placement of buttons

### ISSUE THREAD

**Issue by philippe-hawi** [Dijji/CustomWindowsProperties#6]
Say I have 2 installed custom properties, prop1 and prop2. If I select prop1 in the top left box, and prop2 in the bottom left box, the uninstall button uninstalls prop2. Perhaps this is not the most intuitive way to do it. A couple of times, I would select the property in the top left box and click on uninstall only to find that I uninstalled some selected property in the bottom left box.

**Comment by Dijji** [comment]
You're right. One thinks of install and uninstall as a pair of operations, the same thing, but actually, each operates on a different thing.

There is logic there: you can only install what isn't installed, which means what shows in the top left, and uninstall what is installed, which means that which shows in the bottom left.

But this is not at all clearly signalled. Can you think of any better way of organising the UI to make it clearer?

**Comment by philippe-hawi** [comment]
As a quick first picture in my mind, either:
- separate the buttons by moving the relevant buttons to the top or bottom left box
- move these functionalities to a context menu (right click on a property to see the available actions)
- make relevant buttons appear next to each property when it is selected. In this case, also add the ability to install/uninstall properties from the top left box. If a property is installed, show an uninstall button (and vice versa). Also, perhaps group or color code or use bold face to distinguish between installed custom properties and not-installed custom properties.

Also, perhaps add titles to the boxes (e.g. local properties, Installed properties).

This is a quick response. I'll give it more thought and comment again.

**Comment by Dijji** [comment]
Thank you for the further input, I will to give it some thought as well.

In the meanwhile, I have just posted 0.2, with an experiment where I add property names to the buttons so you know what they are going to affect. Just another idea.
